operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -66,6 +66,26 @@ The basic outline for getting started with contributions is provided in the [REA
 
 Once complete, submit your Pull Request \(PR\). Ensure that you assign appropriate approvers and labels. You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review) - it applies to various types of contributions including documentation.
 
+### Submitting Feature Documentation (PMs)
+
+Feature documentation is a joint effort between Product Managers and Technical Writers. In the same way that we want to empower everyone to contribute to our documentation, Product Managers are encouraged to write MVP documentation for their product/feature. 
+
+**Note**: Due to the cadence of the release cycle, feature documentation needs to be complete and submitted as per the process below in order to ensure it's included in the documentation update. 
+
+The supplied content can be provided informally, in bullet points, or rough notes in a Google Doc and refinements are made collaboratively. This is a guideline of what MVP feature documentation can include:
```

Comment:
"can include" or "should include"

Reply:
I think "can" works better as not all updates include config settings/examples, or troubleshooting. That said, the list mentions relevancy so this can be changed.